src/OnlineRetailPortal.Core/Models/Product.cs

```diff
@@ -28,6 +28,8 @@ public Product(Price price, string sellerId, string name)
             SellerId = sellerId;
             Name = name;
         }
+        public Product ()
```

Comment:
remove if it does contain any implementation

Reply:
we are using it , by constructor overloading